backend/scripts/test_activity_feed.sh

```
@@ -0,0 +1,416 @@
+    if [ -z "$users_raw" ]; then
+        echo -e "${RED}✗ No users found in database. Please create users first.${NC}"
+        exit 1
+    fi
+
+    echo ""
+    echo "Available users:"
+    local i=1
+    declare -a USER_IDS USER_NAMES
+    while IFS='|' read -r uid name username; do
+        printf "  %2d) %-20s  @%-20s  %s\n" "$i" "$name" "$username" "$uid"
+        USER_IDS[$i]="$uid"
```

Shellcheck (0.11.0)

[style] $/${} is unnecessary on arithmetic variables.

(SC2004)